erpnext/regional/doctype/gst_hsn_code/gst_hsn_code.json

```diff
@@ -32,13 +32,37 @@
    "options": "Item Tax"
   }
  ],
- "links": [],
- "modified": "2019-12-29 11:32:06.849631",
+ "modified": "2020-01-27 16:20:09.277395",
  "modified_by": "Administrator",
  "module": "Regional",
  "name": "GST HSN Code",
  "owner": "Administrator",
- "permissions": [],
+ "permissions": [
```

Comment:
Stock Manager and Item Manager Should also have permission for GST HSN CODE since they might need it to while creating new items